OUT
## Observations
Thu Oct 16 21:42:57 CEST 2014

## stick

* not ok

## pump


## downloaded:        0

```
```


## commands session:finished:        0

```
```

howdy! pump runs were NOT OK

### Last send command

```
```
### stats before traceback

```
```
### Traceback

```
echo "OUT" | tee logs/explain.log
. ./explain-brief.sh | tee -a logs/explain.log
explain_markdown


#####
# EOF
Thu Oct 16 21:42:56 CEST 2014
```
INFO:__main__:howdy! I'm going to take a look at your carelink usb stick.
Traceback (most recent call last):
  File "decocare/stick.py", line 883, in <module>
    stick = Stick(link.Link(port))
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.09 real         0.07 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump.
--
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.09 real         0.07 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump.
Traceback (most recent call last):
  File "decocare/session.py", line 122, in <module>
    stick = stick.Stick(link.Link(port, timeout=.200))
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.09 real         0.06 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump and grab lots of info.
--
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.09 real         0.06 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump and grab lots of info.
Traceback (most recent call last):
  File "decocare/commands.py", line 1481, in <module>
    stick = stick.Stick(link.Link(port, timeout=.400))
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.20 real         0.17 user         0.02 sys
```
Was there an ACK ERROR?
### DIAGNOSE CRC
```
INFO:__main__:howdy! I'm going to take a look at your carelink usb stick.
--
--
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.20 real         0.17 user         0.02 sys
```
Was there an ACK ERROR?
### DIAGNOSE CRC
```
INFO:__main__:howdy! I'm going to take a look at your carelink usb stick.
Traceback (most recent call last):
  File "decocare/stick.py", line 883, in <module>
    stick = Stick(link.Link(port))
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.10 real         0.07 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump download something info.
--
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.10 real         0.07 user         0.02 sys
```
```
INFO:__main__:howdy! I'm going to take a look at your pump download something info.
Traceback (most recent call last):
  File "decocare/download.py", line 78, in <module>
    stick = stick.Stick(link.Link(port, timeout=.400))
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/Users/mdijkstra/Documents/git/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "build/bdist.macosx-10.9-intel/egg/serial/serialutil.py", line 282, in __init__
  File "build/bdist.macosx-10.9-intel/egg/serial/serialposix.py", line 289, in open
OSError: [Errno 2] No such file or directory: '/dev/ttys004'
        0.09 real         0.06 user         0.02 sys
```
```
* NO CRC ERROR FOUND
* no nak found
* NOT A CLEAN RUN
